lidar_multiclass/data/loading.py

```diff
@@ -97,22 +106,27 @@ def split_and_save(self, filepath: str, output_subdir_path: str) -> None:
         Args:
             filepath (str): input LAS file
             output_subdir_path (str): output directory to save splitted `.data` objects.
+
         """
+        range_by_axis = np.arange(
+            self.input_tile_width_meters // self.subtile_width_meters + 1
+        )
+
         data = self.load_las(filepath)
         idx = 0
-        for _ in tqdm(self.range_by_axis):
+        for _ in tqdm(range_by_axis):
             if len(data.pos) == 0:
                 break
             data_x_band = self._extract_by_x(data)
-            for _ in self.range_by_axis:
+            for _ in range_by_axis:
                 if len(data_x_band.pos) == 0:
```

Comment:
même commentaire, peut-être : 
if not data_x_band.pos :

Reply:
cf. https://github.com/IGNF/lidar-deep-segmentation/pull/14#discussion_r874515636